pyproject.toml

```diff
@@ -0,0 +1,49 @@
+[build-system]
+requires = ["setuptools>=45", "wheel", "setuptools_scm[toml]>=6.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "basyx-python-sdk"
+version = "1.0.0"
+description = "The Eclipse BaSyx Python SDK, an implementation of the Asset Administration Shell for Industry 4.0 systems"
+authors = [
+    { name = "The Eclipse BaSyx Authors", email = "admins@iat.rwth-aachen.de" }
+]
+readme = "README.md"
+license = { file = "LICENSE" }
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+    "Development Status :: 5 - Production/Stable"
+]
+requires-python = ">=3.8"
+dependencies = [
+    "python-dateutil>=2.8,<3",
+    "lxml>=4.2,<5",
+    "urllib3>=1.26,<2.0",
+    "pyecma376-2>=0.2.4"
+]
+
+[project.optional-dependencies]
+dev = [
+    "mypy",
+    "pycodestyle",
+    "codeblocks",
+    "coverage",
+    # You may want to keep other necessary development dependencies here
```

Comment:
This comment is unnecessary as it is obvious, I'd remove it. 

A better comment would be something above the `dev = [` line that explains when a user would want to install the `dev` dependencies. For now, I think it's fine to just remove this comment.